### PR TITLE
Level order traversal in C++

### PR DESCRIPTION
**Pull request by hritikgupta92**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Level Order Traversal is the traversal of the nodes in a binary tree in a BFS( Breadth First Search) order 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    cout<<"Level Order Traversal for the following Tree will be:"<<endl;
+     
+    levelOrder(root);
+     
+    return 0;
+
+}
+ 
+
+
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Add sample I/O and code complexity

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+              q.push(t->RightPtr);
+         
+              q.pop();
+         }
+      
+ }
+  
+int main()
+{ 
+    //  Input Values here
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Remove the redundant comment

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    
+      Sample Output 1:
+      Level Order Traversal for the following Tree will be   :  1 2 3 4 5 6
+	
+	
+      Sample Input 2:
+      Enter the values of nodes of the tree  :  3 4 6 -1 -1 7 -1 -1 5 -1 -1  
+
+      Sample Output 2:
+      Level Order Traversal for the following Tree will be   :  3 4 5 6 7
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Remove this

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp [hunk 4]
```diff
@@ -0,0 +1,115 @@
+/*
+    Level Order Traversal
+    
+	Given a binary tree with unique values,print all the elements in 
+      a level Order i.e in a BFS( Breadth First Search) order.
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Add a bit more information

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp (hunk 1 above)
Comment (hritikgupta92) [reply C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Done

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp (hunk 3 above)
Comment (hritikgupta92) [reply C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Ok I have removed this now

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp (hunk 4 above)
Comment (hritikgupta92) [reply C-Plus-Plus/ds/LevelOrderTraversal.cpp]: done

File: C-Plus-Plus/ds/LevelOrderTraversal.cpp (hunk 2 above)
Comment (hritikgupta92) [reply C-Plus-Plus/ds/LevelOrderTraversal.cpp]: Done

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**